tests/integration/api/test_rulebook.py

```diff
@@ -352,6 +352,34 @@ def test_list_audit_rules_ordering_activation_name(
     )
 
 
+@pytest.mark.django_db
+def test_list_audit_rules_filter_activation_instance_name(
+    audit_rule_1: models.AuditRule,
+    audit_rule_2: models.AuditRule,
+    admin_client: APIClient,
+):
+    filter_name = audit_rule_1.activation_instance.name
+    response = admin_client.get(
+        f"{api_url_v1}/audit-rules/?activation_instance__name={filter_name}"
```

Comment:
@Dostonbek1 Since we have removed the Audit Action and Audit Event filter should we have a test case for that. Also doesn't the DAB rest filter support AND and OR combinations. should we have a test based on that. To pick a specific rule name under an Activation Instance. 
```
f"{api_url_v1}/audit-rules/?activation_instance__name={filter_name} AND rule_name=R2" something like this.

Reply:
@mkanoor We are removing our custom filtering from Audit Action and Audit Event but the way we are implementing DAB filters is that it automatically applies to all views that do not have filter class specified, so Audit Action and Audit Event are using DAB filters right now with these changes. And we do have existing tests for these; check tests starting on line 418 on this file.